cve_bin_tool/html/templates/mystylesheet.html

```diff
@@ -0,0 +1,88 @@
+<style>
+    body{
```

Comment:
I think mystylesheet should probably actually be a stylesheet, and just put the <style> tags in where it's imported.